src/me/EtienneDx/RealEstate/Config.java

```diff
@@ -212,4 +220,105 @@ public void loadConfig()
             RealEstate.instance.log.info("Unable to write to the configuration file at \"" + this.configFilePath + "\"");
         }
     }*/
+
+    public void loadMessages() {
+        Messages[] messageIDs = Messages.values();
+        this.messages = new String[Messages.values().length];
+
+        HashMap<String, CustomizableMessage> defaults = new HashMap<String, CustomizableMessage>();
+        // initialize defaults
+        this.addDefault(defaults, Messages.NoTransactionFound, "No transaction found at your location!", null);
+        this.addDefault(defaults, Messages.PageMustBePositive, "Page must be a positive option!", null);
+        this.addDefault(defaults, Messages.PageNotExists, "This page does not exist!", null);
+        this.addDefault(defaults, Messages.RenewRentNow, "Automatic renew is now $a{0} $bfor this {1}", "0: the status; 1: a claim type");
+        this.addDefault(defaults, Messages.RenewRentCurrently, "Automatic renew is currently $a{0} $bfor this {1}", "0: the status; 1: a claim type");

```

Comment:
Wouldn't it be clearer to put the message color here by default, and not in the send message?
Cause here there might be inconsistencies if the server admin replaces the $a and $b by hand.

```suggestion
        this.addDefault(defaults, Messages.NoTransactionFound, "$cNo transaction found at your location!", null);
        this.addDefault(defaults, Messages.PageMustBePositive, "$cPage must be a positive option!", null);
        this.addDefault(defaults, Messages.PageNotExists, "$cThis page does not exist!", null);
        this.addDefault(defaults, Messages.RenewRentNow, "$bAutomatic renew is now $a{0} $bfor this {1}", "0: the status; 1: a claim type");
        this.addDefault(defaults, Messages.RenewRentCurrently, "$bAutomatic renew is currently $a{0} $bfor this {1}", "0: the status; 1: a claim type");

```